### DISCUSSION TITLE
Delete image attached to the post

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#541] [Feature Requests]
> This is a fantastic plugin. Posts can really clutter a website and especially if they are old and about events that have expired. Not everything needs to be archived. One thing that is left when you delete a post is the Featured Image. It would be great if there was a setting to delete the image along with the post. This would help to keep the Image Library cleaner and not end up with thousands of old unused images. Thank you again for such an effective plugin.

Original request: https://wordpress.org/support/topic/delete-media-library-images/

**Comment by rizaardiyanto1412** [comment]
Another request: https://wordpress.org/support/topic/add-the-option-to-delete-all-images-from-the-post-or-page/

**Comment by andergmartins** [comment]
Probably related to #628
